contents/newsletter/hiring-at-posthog-lessons.md

```diff
@@ -0,0 +1,161 @@
+
+Maybe not, but many of the best people at PostHog worked at or founded companies that just didn’t work. We find this cultivates a degree of grit and optimism that’s invaluable in a startup, not to mention experience of all the mistakes they made along the way.
+
+In contrast, someone who’s worked exclusively at FAANG-type companies c.2012-2022, or joined startups after they’d achieved product-market fit, has (maybe) had an easy life.
+
+**Why is this important?** Working at a startup, especially before product-market fit, is totally different from a bluechip, or a startup that’s already (mostly) got stuff figured out. Don’t assume people will work just because their CV looks right.
+
+## Good reads about hiring 📚
+
+[How to hire your first engineer](https://www.ycombinator.com/library/4H-how-to-hire-your-first-engineer/)** – Haj Taggar**
+”Candidates think differently to investors and you'll need to tweak the message that worked for your fundraise e.g. candidates will think less about your market size and more about your most interesting product challenges.”
+
+[Myths (and truths) I've learned from 10 years of startup hiring](https://posthog.com/founders/early-stage-startup-hiring-strategy)** – Charles Cook**
+”I've managed and hired for teams across engineering, ops, people, marketing, and design, interviewing some 800 people in the process. These are the destructive myths and important truths I've learned over that time.”
+
+**[The SaaSy guide to hiring](https://staysaasy.com/hiring/2022/09/23/saasy-guide-to-hiring-how-to-hire.html/) – Stay SaaSy**
+“If you’re a tech company, it’s worth your while to refine your recruiting process as if you were a company like Toyota or Apple refining their supply chain. Recruiting should be a priority for the smartest and most driven people at your company, rather than a backwater activity that gets passed around like a hot potato.”
+
+**[PostHog hiring handbook](https://posthog.com/handbook/people/hiring-process/) – PostHog**
+“For everyone on the PostHog team meeting a candidate, ask yourself – will this person raise the bar at PostHog? The answer should be yes if we want to hire them.”
+
+**[Why do companies over-hire?](https://waysofworkingcollection.substack.com/p/why-do-companies-over-hire/) – Josephine Conneely**
+“If [hiring for growth] ask yourself, what would the leanest version of hiring look like? What is the expected output difference versus current hiring plans? It may be possible to achieve most of your growth goals with a smaller team.”
+
+**[What we learned about hiring from our first five employees](https://posthog.com/blog/posthog-first-five) – Andy Vandervell**
+“While it's tempting to search for "that hire" who will magically transform your product and company, this is a fool's errand. Successful companies are built on collective strength, which is why talent compounds is one of our core values.”
```

Comment:
```suggestion
[How to hire your first engineer](https://www.ycombinator.com/library/4H-how-to-hire-your-first-engineer/)** – Haj Taggar:** ”Candidates think differently to investors and you'll need to tweak the message that worked for your fundraise e.g. candidates will think less about your market size and more about your most interesting product challenges.”

[Myths (and truths) I've learned from 10 years of startup hiring](https://posthog.com/founders/early-stage-startup-hiring-strategy)** – Charles Cook:** ”I've managed and hired for teams across engineering, ops, people, marketing, and design, interviewing some 800 people in the process. These are the destructive myths and important truths I've learned over that time.”

**[The SaaSy guide to hiring](https://staysaasy.com/hiring/2022/09/23/saasy-guide-to-hiring-how-to-hire.html/) – Stay SaaSy:** “If you’re a tech company, it’s worth your while to refine your recruiting process as if you were a company like Toyota or Apple refining their supply chain. Recruiting should be a priority for the smartest and most driven people at your company, rather than a backwater activity that gets passed around like a hot potato.”

**[PostHog hiring handbook](https://posthog.com/handbook/people/hiring-process/) – PostHog:** “For everyone on the PostHog team meeting a candidate, ask yourself – will this person raise the bar at PostHog? The answer should be yes if we want to hire them.”

**[Why do companies over-hire?](https://waysofworkingcollection.substack.com/p/why-do-companies-over-hire/) – Josephine Conneely:** “If [hiring for growth] ask yourself, what would the leanest version of hiring look like? What is the expected output difference versus current hiring plans? It may be possible to achieve most of your growth goals with a smaller team.”

**[What we learned about hiring from our first five employees](https://posthog.com/blog/posthog-first-five) – Andy Vandervell:** “While it's tempting to search for "that hire" who will magically transform your product and company, this is a fool's errand. Successful companies are built on collective strength, which is why talent compounds is one of our core values.”
```